### PR TITLE
Hazel's Changes for PDOCS-1019

### PR DESCRIPTION
**Pull request by dbrian57**
This PR addresses PDOCS-1019. Just porting some changes for Hazel until I better document how PDOCs can contribute to the spec.

### CODE REVIEW

File: specification/resources/cdn/models/cdn_endpoint.yml [hunk 1]
```diff
@@ -29,7 +29,8 @@ properties:
     type: integer
     example: 3600
```
Comment (andrewsomething) [inline specification/resources/cdn/models/cdn_endpoint.yml:30]:
```suggestion
    example: 3600
    enum:
    - 60
    - 600
    - 3600
    - 86400
    - 604800
    default: 3600
```

File: specification/resources/cdn/models/update_endpoint.yml [hunk 2]
```diff
@@ -5,7 +5,8 @@ properties:
     type: integer
     example: 3600
```
Comment (andrewsomething) [inline specification/resources/cdn/models/update_endpoint.yml:6]:
```suggestion
    example: 3600
    enum:
    - 60
    - 600
    - 3600
    - 86400
    - 604800
    default: 3600
```

**Review by andrewsomething — CHANGES_REQUESTED** [review]
Since this looks like an enum, it would be great to enumerate the possible values as well.

https://swagger.io/docs/specification/data-models/enums/

**Comment by dbrian57** [comment]
Thanks, @andrewsomething!

**Comment by andrewsomething** [comment]
!deploy

**Review by andrewsomething — APPROVED** [review]
:ship:  LGTM!

**Pull request merged**